lib/services/plugins/nativescript-project-plugins-service.ts

```diff
@@ -220,7 +220,7 @@ export class NativeScriptProjectPluginsService extends NpmPluginsServiceBase imp
 		}).future<IBasicPluginInformation>()();
 	}
 
-	protected fetchPluginBasicInformationCore(pathToInstalledPlugin: string, pluginData?: ILocalPluginData): IFuture<IBasicPluginInformation> {
+	protected fetchPluginBasicInformationCore(pathToInstalledPlugin: string, pluginData?: ILocalPluginData, options: NpmPlugins.IFetchLocalPluginOptions = { useOriginalPluginDirectory: false }): IFuture<IBasicPluginInformation> {
```

Comment:
You can set the `options` argument as non-mandatory only without setting a default value. The `installLocalPlugin` will set the default value if needed.